actors/builtin/storage_power/storage_power_actor.go

```diff
@@ -37,14 +39,41 @@ type StoragePowerActor struct{}
 // Actor methods
 ////////////////////////////////////////////////////////////////////////////////
 
-func (a *StoragePowerActor) AddBalance(rt Runtime, minerAddr addr.Address) *adt.EmptyValue {
-	builtin.RT_MinerEntry_ValidateCaller_DetermineFundsLocation(rt, minerAddr, builtin.MinerEntrySpec_MinerOnly)
+func (a *StoragePowerActor) Constructor(rt Runtime, _ *adt.EmptyValue) *adt.EmptyValue {
```

Comment:
To keep method signatures (func (Runtime, CBORMarshal)) the same I've introduced this parameter, any objection?

Reply:
Beware conflict with #68